bindings/java/src/org/sleuthkit/datamodel/OsAccountManager.java

```diff
@@ -824,22 +814,21 @@ private OsAccount osAccountFromResultSet(ResultSet rs, OsAccountRealm realm) thr
 	/**
 	 * Created an account signature for an OS Account. This signature is simply
 	 * to prevent duplicate accounts from being created. Signature is set to:
-	 * realmId/uniqueId: if the account has a uniqueId, otherwise
-	 * realmId/loginName: if the account has a login name.
+	 * uniqueId: if the account has a uniqueId, otherwise
+	 * loginName: if the account has a login name.
 	 *
-	 * @param realm     Account realm
 	 * @param uniqueId  Unique id.
 	 * @param loginName Login name.
 	 *
 	 * @return Account signature.
 	 */
-	private String makeAccountSignature(OsAccountRealm realm, String uniqueId,  String loginName) {
+	private String makeAccountSignature(String uniqueId,  String loginName) {
```

Comment:
This could probably be just updateSignature() and refer to the member variables, right?

Reply:
Umm, signature is a member in OsAccount, this is OsAccountManager.  

We have to write the signature column in the database before we even create a OsAccount object.

It could certainly use a different name now - since there is no more "making"